### ISSUE TITLE
C3jenkins disk full ?

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/infrastructure#2750]
Noticed all the Skara mirror jobs are failing due to no disk space errors..
Looks like C3jenkins disk is full?
```
Filesystem      Size  Used Avail Use% Mounted on
udev             16G     0   16G   0% /dev
tmpfs           3.2G  1.7M  3.2G   1% /run
/dev/sda3       439G  430G     0 100% /
```

**Comment by sxa** [comment]
System has a 512Gb FS which contains AWX in docker images (~128Gb) and a backup jenkins setup, which can likely be removed now (~75Gb)

In the jenkins workspace, 115Gb is used by `build-scripts`. I've reduced that to 108Gb by removing `@n` directories from the workspace for now.

We've got 88Gb available which should allow things to continue. This will, of course, be a candidate for monitoring/alerting by Nagios #1716 

Closing as the immediate problem is resolved.
